src/node.rs

```diff
@@ -184,6 +184,11 @@ impl Node {
         self.machine.current().close_group(name, count)
     }
 
+    /// Returns the connection information of all the current section elders.
+    pub fn our_elders_info<'a>(&'a self) -> Option<Box<dyn Iterator<Item = &P2pNode> + 'a>> {
```

Comment:
One more thing, should we have a small test for this?

Reply:
Question: why return `Option<Iterator>` instead of just returning empty iterator in the case we are not elder?

Reply:
As discussed, an empty iterator could be misguiding and indicate that there are no elders vs `Option::None` for and `Option<impl Iterator>` would mean that it's an invalid request or not possible to create this iterator. So leaving it as `Option` for now.

Reply:
No test then? it is probably ok for what is an accessor I guess.